docs/intent.md

```diff
@@ -0,0 +1,41 @@
+# Curriculum intent
+
+This document explains the high-level intent behind our curriculum—the motivations behind decisions and trade-offs we have made, and what we expect learners to get out of it.
+
+## Scope
+
+Our programme has limited time to get learners from understanding the basics to working as professional software developers in a Level 5 apprenticeship. This means we have to make strict decisions regarding what concepts to cover in the curriculum.
+
+Software development is a hugely broad discipline, with hundreds of different programming languages and paradigms to write code within. Our curriculum is specifically focused on writing software for the (World Wide) Web. This means we exclusively use the languages native to the web—HTML, CSS and JavaScript (with a little SQL included as it is the closest thing to a standardised language for data access).
+
+We focus on full-stack web development, which means primarily _application_ code. Our programme is designed for people who want to build web applications—this includes both client-side and server-side code. We do _not_ focus on building native applications, systems programming, hardware, or other lower-level concepts.
+
+## Accessibility
+
+One of Founders and Coders' founding principles is to improve access to the tech industry. Our curriculum is therefore designed to be as accessible as possible. All of the languages and technology we use and recommend are free and open source. All a learner should need to get started is access to a computer and an internet connection. Our resources are explicitly focused on people without an existing technical background, and should never require prior expertise or academic qualifications in computer science.
+
+## Meta learning outcomes
```

Comment:
depends on our target audience for this but I don't think everyone is familiar with the term "meta".

maybe this should be "broader learning outcomes" or something similar